packages/core/test/test_util/common_tests.js

```diff
@@ -0,0 +1,131 @@
+/*
```

Comment:
I would prefer a file name without `_tests`. This sounds like it is a set of tests instead of a set of utility methods. How about something along the lines of `common_verifications`, `common_checks` or maybe `span_verifications` (so far we only have verifyXxxSpan methods here).